cbmc/patches/0001-Patch-MQTT-library-code-for-CBMC-Unsubscribe-proofs.patch

```diff
@@ -68,19 +68,6 @@ index 5afce54..0e4d923 100644
          if( pOperation != NULL )
          {
              IotMqtt_FreeOperation( pOperation );
```

Comment:
Why remove this ?

Reply:
I moved this modification to the patch `0001-Replace-limits-to-improve-coverage-on-IotMqtt_Connect.patch`.

Reply:
Let's talk about this.

Reply:
Use a name like replace limits from internal constant definitions

Reply:
Done.